lib/node_modules/@stdlib/stats/base/dsmean/test/test.dsmean.js

```diff
@@ -125,47 +121,40 @@ tape( 'the function supports a negative `stride` parameter', function test( t )
 		2.0
 	]);
 
-	N = floor( x.length / 2 );
-	v = dsmean( N, x, -2 );
+	v = dsmean( 4, x, -2, 6 );
```

Comment:
Be careful copying and pasting. You copied over the `ndarray` calls, but the main export doesn't support an offset parameter. You need to go back through and remove any offset arguments in this file.